vtexdocs/devportal
# Feature/MDX Images

#### What is the purpose of this pull request?

To add the script for downloading images from MDX files.

Documented [here](https://www.notion.so/vtexhandbook/Gerenciar-imagens-dentro-dos-MDXs-2d980e356f3245d5809ae03033ef4866)

#### What problem is this solving?

The images inside MDX files should be stored locally to improve performance.

#### How should this be manually tested?

Go to [this page](https://deploy-preview-68--elated-hoover-5c29bf.netlify.app/docs/api-guides/developing-an-app) and compare it with [this one](https://elated-hoover-5c29bf.netlify.app/docs/api-guides/developing-an-app): check if all the images are still there. You can also try running the app locally and verifying that the images are downloaded to `/public`

#### Screenshots or example usage

#### Types of changes
